flixel/FlxSprite.hx

```diff
@@ -65,9 +65,8 @@ class FlxSprite extends FlxObject
 	
 	/**
 	 * Controls whether the object is smoothed when rotated, affects performance.
-	 * It's better to use `smoothing` property instead, since `antialiasing` is deprecated.
 	 */
-	@:deprecated
+	@:deprecated("It's better to use `smoothing` property instead.")
```

Comment:
Sorry to be so nit-picky, but "It's better to" to me implies that they somehow do different things, when it's really just been renamed.